### ISSUE TITLE
cannot create new notebook '_xsrf' argument missing from POST jupyter/datascience-notebook

### ISSUE THREAD

**Issue by jdcohen220** [jupyter/docker-stacks#741]
I am using the latest `jupyter/datascience-notebook` image for the first time, and was able to boot up the container with `docker run -p 8888:8888 jupyter/datascience-notebook `, however when I access `Home` from the browser and try to open a new notebook I get the following error: 

> Creating Notebook Failed
> An error occurred while creating a new notebook.
> '_xsrf' argument missing from POST

Complete terminal output from an example session:
```
jcohen-osx:~ jcohen$ docker run -p 8888:8888 jupyter/datascience-notebook
Container must be run with group "root" to update passwd file
Executing the command: jupyter notebook
[I 21:33:36.915 NotebookApp] Writing notebook server cookie secret to /home/jovyan/.local/share/jupyter/runtime/notebook_cookie_secret
[I 21:33:37.203 NotebookApp] JupyterLab extension loaded from /opt/conda/lib/python3.6/site-packages/jupyterlab
[I 21:33:37.203 NotebookApp] JupyterLab application directory is /opt/conda/share/jupyter/lab
[I 21:33:37.213 NotebookApp] Serving notebooks from local directory: /home/jovyan
[I 21:33:37.214 NotebookApp] The Jupyter Notebook is running at:
[I 21:33:37.214 NotebookApp] http://(63a7d5be5be2 or 127.0.0.1):8888/?token=b87ed495caa6c986a51bf6e7de3391ff01e51a93a20436ef
[I 21:33:37.215 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 21:33:37.216 NotebookApp]

    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://(63a7d5be5be2 or 127.0.0.1):8888/?token=b87ed495caa6c986a51bf6e7de3391ff01e51a93a20436ef
[W 21:33:54.669 NotebookApp] Clearing invalid/expired login cookie username-127-0-0-1-8888
[W 21:33:54.670 NotebookApp] Forbidden
[W 21:33:54.671 NotebookApp] 403 GET /api/terminals?_=1540848600493 (172.17.0.1) 3.02ms referer=http://127.0.0.1:8888/tree
[W 21:33:54.676 NotebookApp] Clearing invalid/expired login cookie username-127-0-0-1-8888
[W 21:33:54.680 NotebookApp] Forbidden
[W 21:33:54.681 NotebookApp] 403 GET /api/sessions?_=1540848600492 (172.17.0.1) 6.78ms referer=http://127.0.0.1:8888/tree
[I 21:34:04.809 NotebookApp] 302 GET /?token=b87ed495caa6c986a51bf6e7de3391ff01e51a93a20436ef (172.17.0.1) 3.15ms
[W 21:34:08.687 NotebookApp] 403 POST /api/contents (172.17.0.1): '_xsrf' argument missing from POST
[W 21:34:08.688 NotebookApp] '_xsrf' argument missing from POST
[W 21:34:08.690 NotebookApp] 403 POST /api/contents (172.17.0.1) 5.62ms referer=http://127.0.0.1:8888/tree
```

**Comment by ericdill** [comment]
Hi @jdcohen220,

I just tried this and everything worked as expected on my ec2 instance. I've got a few questions to help debug this issue.

1. Can you report the output of `docker images`?
2. What is your host OS for docker?
3. Is this repeatable on your machine?
4. What browser and browser version are you using?
5. Does this behavior go away if you try a different browser or if you use incognito/private mode in your browser of choice?

**Comment by jdcohen220** [comment]
1) Docker images:
```
jcohen-osx:~ jcohen$ docker images | grep jupy
jupyter/datascience-notebook                                                                                                              latest              ae4c1f6633af        4 days ago          6.03GB
```

1) I am using `macOS Mojave==10.14`
3) Yes I am able to repeat this behavior on my machine.
4) This behavior manifests in both `Chrome==69.0.3497.100` and `Brave==0.25.2`.
5) I was able to make the behavior go away by using either `localhost` or the hash-looking hostname, not the `127.0.0.1`.

**Comment by parente** [comment]
@jdcohen220 It sounds silly, but I've seen the above behavior when I've accidentally run another Jupyter Notebook server (or other server) directly on my Mac on port 8888. If you shut down the container and visit http://127.0.0.1:8888 do you see anything else using that port?

**Comment by jdcohen220** [comment]
@parente I double checked that the container was shutdown, and that no processes were running from that port or host (using browser/ping/lsof), and everything worked as expected on Chrome. I cannot say for certain I did not have any other servers running during the initial problem I had yesterday, since I did not even think to check.

I reproduced the same steps Brave, and it still does not work with the 127.0.0.1 address. Using `localhost` works fine, however. 

Probably not a huge deal -- I am planning on adding it to my `"list-of-things-Brave-doesn't-play-nice-with"`, and switching back to Chrome as default browser. Thanks for helping out!
